doc/api/tls.md

```diff
@@ -357,15 +357,15 @@ the default configuration. If these clients _must_ be supported, the
 [TLS recommendations][] may offer a compatible cipher suite. For more details
 on the format, see the OpenSSL [cipher list format][] documentation.
 
-There are only 5 TLSv1.3 cipher suites:
+There are only five TLSv1.3 cipher suites:
```

Comment:
Optional nit: _only_ seems unnecessary?

```suggestion
There are five TLSv1.3 cipher suites:
```

Reply:
The small number of different cipher suites is a key difference between TLS 1.3 and TLS 1.2, since the older protocol defines at least 20 or so. The new protocol has been simplified greatly in this regard. I personally think emphasizing the difference makes sense here :)